src/common-principles.md

```diff
@@ -238,10 +238,14 @@ distinguish partial results from the raw data and share the latter.
 See [Storage of derived datasets](#storage-of-derived-datasets) for more on
 organizing derivatives.
 
-Similar rules apply to source data, which is defined as data before
-harmonization, reconstruction, and/or file format conversion (for example, E-Prime event logs or DICOM files).
-Storing actual source files with the data is preferred over links to
-external source repositories to maximize long term preservation,
+Similar rules apply to source data, which is defined as data
+before harmonization, reconstruction, and/or file format conversion
+(for example, E-Prime event logs or DICOM files).
+Retaining the source data is especially valuable
+in a case when conversion fails to preserve crucial metadata
+unique to specific acquisition setup.
+Storing actual source files with the data is preferred over links
+to external source repositories to maximize long term preservation,
```

Comment:
You may be onto something. Are you ok if we put that into another issue?

I have bad track record of turning cute small PRs into monsstruosity that are 500s lines long that no one wants to review. Am trying to work on it. 😉

Reply:
Agreed. Let's not let the perfect be the enemy of the good, so if there isn't a specific complaint, let's let an improvement come when someone's ready to write text. The `exclude-from-changelog` label is useful if we're worried about polluting the changelog.

Just to add more material for reflection: BIDS doesn't seem to know whether it wants to be a reference, a tutorial, or an explainer. At the size it has become, I think it could benefit from refactoring into explicit parts, possibly following [Diataxis](https://diataxis.fr/).

Reply:
Sounds reasonable to me